packages/hash/frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Box,
+  // eslint-disable-next-line no-restricted-imports
+  Button as MuiButton,
+  ButtonProps as MuiButtonProps,
+} from "@mui/material";
+import { FC, forwardRef } from "react";
+
```

Comment:
Pulled this over from BP repo. Actual styles/variants will be implemented in this [internal link](https://app.asana.com/0/1201864704093982/1201916223398333)